### ISSUE TITLE
Why does identify_axis  return [1,2] or [1,2,3]?

### ISSUE THREAD

**Issue by JohnMasoner** [mlyg/mixed-focal-loss#11]
sorry, i dont understand why does identify_axis  return [1,2] or [1,2,3], not [0,1] or [0,1,2].

when i try to run the loss function. the input shape is [B, N, C, W, H], it only calculate the [N, C, W].

I am looking forward to your reply.

**Comment by mlyg** [comment]
Thanks for your question JohnMasoner.

The main point of the identify_axis function is to return the axes belonging to the pixel dimensions i.e. if you are using 2D images then it will return the H and W axes, and if you are using 3D images then it will return the H, W and Depth axes. 

The problem seems to be in your input shape. I am confused with your input shape because 'B' which is the Batch size, and N which is the number of images in the batch, refer to the same thing. Normally, for 2D images, a Tensorflow tensor would be in the format 'NHWC', where the channels are 'last' (different to PyTorch which uses the 'NCHW ' format).

Let me know if you need any more help.

Michael

**Comment by JohnMasoner** [comment]
Thanks for your answer Michael,

> The main point of the identify_axis function is to return the axes belonging to the pixel dimensions i.e. if you are using 2D images then it will return the H and W axes, and if you are using 3D images then it will return the H, W and Depth axes.

you told me, my input datas is different from the loss function. i will change my code.

thank you so much for the answer

Mason
